actian_jdbc/connectionFields.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<connection-fields>
+  <field name="server" label="Server" value-type="string" category="endpoint" >
+    <validation-rule reg-exp="^(([a-zA-Z]|[a-zA-Z][a-zA-Z0-9\-]*[a-zA-Z0-9])\.)*([A-Za-z]|[A-Za-z][A-Za-z0-9\-]*[A-Za-z0-9])$"/>

```

Comment:
this may need a comment (if possible).

It looks like for AV we are pushing numerics so that needs to work as well.

Reply:
With the addition of new file connectionFields.xml, the JDBC taco code now supports both numeric ports (`21071`, `27839`, etc.) and aliases (e.g. `II7`).

I believe the input validation via regular expressions in `connectionFields.xml` could also be defined for the ODBC taco, but as we learned in https://github.com/ActianCorp/actian_tableau_connector/issues/6, there are some differences between connection dialog code [v1](https://tableau.github.io/connector-plugin-sdk/docs/ui) and [v2](https://tableau.github.io/connector-plugin-sdk/docs/mcd). I need to investigate further to understand these differences and how/if the changes apply to both JDBC and ODBC tacos. At a minimum, it would require some rework of the ODBC taco code to support port aliases.  Could that wait until a future release?

Reply:
waiting sounds good, a doc comment would be a good idea (if supported)